src/documents/activities/InclusiveDesignMapping.html.md.handlebars

```diff
@@ -0,0 +1,70 @@
+---
+title: Inclusive Design Mapping
+layout: content
+category: Activities
+---
+
+This activity breaks down the [User States and Contexts Tool](/tools/UserStatesAndContexts.html) into modular pieces (the individual wedges, or "facets") and actionable steps. It is also a re-thinking of the original tool.
+
+Here, <strong>user states are reframed as functional requirements of a product or service</strong>. Deficit language or disability classifications that are functionally ambiguous are avoided.
+
+Needs that are commonly met are placed in the middle, and needs that are not well met are placed toward the edge. This demonstrates that the needs of users "on the edge" are greater in number, as well as the phenomena that by meeting edge needs, you also meet the mainstream needs of the middle.
+
+In this activity, the states and contexts facets can be swapped in and out depending on the context and the design challenge. Facets can also be broken down into more specific considerations. For example, the facet of cognitive access might include memory, focus and other considerations. 
+
+This activity is intended to be used in the context of co-design where a diverse set of co-designers, including people who presently have difficulty using the design, are active participants in the design process. Co-designers are able to create their own facets if they can't find a desired state or context within the provided set.
+
+<ul>
+<li>The exercise is about <strong>stretching and reaching</strong>: to include people who are currently at the edges and who are therefore usually excluded from the design process.</li>
+<li>It is also about <strong>balance and flexibility</strong>: creating a design that does not compromise the experience of one person to make room for the requirements of another.</li>
+</ul> 
+
+## Step 1
+Choose the facets that are relevant to the needs, preferences and the context you are designing for or critiquing.
+<figure>
+[![A diagram showing individual facets of the Inclusive Design Mapping Tool](/images/user states & contexts-modular-01.png)](/images/user states & contexts-modular-01.png)
```

Comment:
Can you rename the file of the image to use Heads-up camel case (no spaces and no extra punctuation) and update this link? This makes the URLs much cleaner.